### DISCUSSION TITLE
Logs are not showing In Dashboard UI

### DISCUSSION THREAD

**Discussion by AjinThankachan** [apache/skywalking#8276] [Q&A]
Hi ,
We are successfully installed and monitored one Java application using Skywalking for testing purpose. In the dashboard we are unable to generate Logs. Can you please help to generate logs of the application into the dashboard? Is there anything else that needs to be configured? Thanks for your support.
Skywalking Version: 8.9
Skywalking Java Agent: 8.8
Java Application :https://github.com/kazuki43zoo/mybatis-spring-boot-jpetstore
Application Running Command : java -javaagent:/home/skywalking-agent/skywalking-agent.jar=-dskywalking.Agent.service_name=Baties -jar target/mybatis-spring-boot-jpetstore-2.0.0-SNAPSHOT.jar

Application Running Screenshot in Shell :
![testimage](https://user-images.githubusercontent.com/25667606/145384863-392ebb1d-ec6d-443d-82ed-d00dde7a3e11.png)
 
Agent Config : [agent.txt](https://github.com/apache/skywalking/files/7684470/agent.txt)
Collector Config : [application.txt](https://github.com/apache/skywalking/files/7684497/application.txt)

![batis](https://user-images.githubusercontent.com/25667606/145384093-b00d3df9-3ebd-49e4-86c8-03858269387d.png)

**Comment by wu-sheng** [comment]
Read quick start doc. Showcase is for your first try. All these are there out of box.

**Reply by AjinThankachan** [reply]
Hi sir,
Thanks for the quick replay.In the website also i am not getting any logs hence i put the query here. We wanted to see how it was populating.

![image](https://user-images.githubusercontent.com/25667606/145386023-b847a15f-96b9-41e5-b3bc-1f8265b1d229.png)

**Reply by wu-sheng** [reply]
Do you choose current duration? Everything works from myside.

![image](https://user-images.githubusercontent.com/5441976/145389375-b9383d82-ed1e-4fbf-aa7d-e7c47f29df9c.jpeg)

**Comment by AjinThankachan** [comment]
Thanks for the quick update. It's showing now. I am not sure the reason i have given 2 days duration initially maybe due to that. Anyway happy to work on this application and extremely happy regarding your support and quicker response.
Thanks a lot.
